Rpackage/R/fitAndScore.R

```diff
@@ -0,0 +1,222 @@
+fit_and_score <- structure(function(target.mat, feature.mat, 
+                                    parameters, n_folds = 3, scorer = NULL, 
+                                    learner = NULL, predict = NULL,
+                                    pruning = TRUE){
+  
+  ### shuffle the data 
+  rand <- sample(nrow(target.mat))
+  target.mat <- target.mat[rand,]
+  feature.mat <- feature.mat[rand,]
+  
+  
+  ### Create n equally size folds
+  folds <- cut(seq(1,nrow(target.mat)), breaks = n_folds, labels = FALSE)
+  
+  ### train and test index for each fold
+  fold_split_idx <- list()
+  
+  ### segment the data into test and train
+  for(i in 1:n_folds){
+    #Segement your data by fold using the which() function 
+    fold_split_idx$test <- rbind(fold_split_idx$test, which(folds == i, arr.ind = TRUE))
+    fold_split_idx$train <- rbind(fold_split_idx$train, which(folds != i, arr.ind = TRUE))
+    
+  }
+  
+  ### fold trees
+  fold_tree <- list()
```

Comment:
Rename the fold_tree, master_tree, etc. variables to fold_model, master_model, etc.

Reply:
done